data/wanderers.txt

```diff
@@ -450,16 +450,16 @@ phrase "friendly wanderer"
 		"any lifeform"
 		"any [identity? consciousness? entity?]"
 	word
-		" "
-		" if they are "
+		", "
```

Comment:
my bad for not being more clear
I meant to redo this word, because "These lands welcome anyone, wanting to..." sounds wrong and maybe is wrong
", if they are" is fine, but ya could rewrite this one to ", as long as they are"